### ISSUE TITLE
[REQ][Java] useAbstractionForFiles=true should return Resource instead of File

### ISSUE THREAD

**Issue by juherr** [OpenAPITools/openapi-generator#10665]
### Is your feature request related to a problem? Please describe.
`useAbstractionForFiles` exists for file param but doesn't change the returned type.

## Describe the solution you'd like
When `useAbstractionForFiles` is `true`, like the param type, the return type should be `Resource` instead of `File`.

**Comment by juherr** [comment]
Related to https://github.com/OpenAPITools/openapi-generator/issues/2486

As a workaround, `Resource` can be forced by configuration:

```xml
                            <typeMappings>
                                <typeMapping>File=org.springframework.core.io.Resource</typeMapping>
                            </typeMappings>
                            <configOptions>
                                <useAbstractionForFiles>true</useAbstractionForFiles>
                            </configOptions>
```
(from https://github.com/OpenAPITools/openapi-generator/issues/2486#issuecomment-717533353)

**Comment by tiffmaelite** [comment]
@juherr  was this fixed by #13146 ?

**Comment by juherr** [comment]
Fixed by #13168

**Comment by juherr** [comment]
@tiffmaelite I don't need it anymore but thanks for the notification. 👍
